src/server/pmix_server_get.c

```diff
@@ -218,11 +218,12 @@ pmix_status_t pmix_server_get(pmix_buffer_t *buf,
     /* if the rank is wildcard, then they are asking for the job-level
      * info for this nspace - provide it */
     if (PMIX_RANK_WILDCARD == rank) {
+        pmix_buffer_t **bptr = &nptr->server->job_info; 
```

Comment:
I see the problem - you declared bptr incorrectly. It should be ```pmix_buffer_t *bptr```, not double-star. Can you please correct?

Reply:
Ah, yes!  ok....